packages/icons/icons.json

```
@@ -2606,161 +2606,161 @@
     },
     {
         "displayName": "Grouped bar chart",
         "iconName": "grouped-bar-chart",
-        "tags": "graph, bar, chart",
+        "tags": "graph, plot",
         "group": "data",
         "codepoint": 61915
     },
     {
         "displayName": "Full stacked chart",
         "iconName": "full-stacked-chart",
-        "tags": "graph, bar, chart",
+        "tags": "graph, bar, cargo, inventory, bricks, wall, chimneys, shipping container, adjust, waterfall",
         "group": "data",
         "codepoint": 61891
     },
     {
         "displayName": "Endorsed",
         "iconName": "endorsed",
-        "tags": "tick, mark, sign, ok, approved, success",
+        "tags": "tick, mark, sign, ok, approved, success, complete, verified, stamp of approval, check, agree",
         "group": "action",
         "codepoint": 61847
     },
     {
         "displayName": "Follower",
         "iconName": "follower",
-        "tags": "person, human, male, female, character, customer, individual, social",
+        "tags": "person, human, male, female, character, customer, individual, social, share, add, send, move, delegate, move person, next",
         "group": "interface",
         "codepoint": 61885
     },
     {
         "displayName": "Following",
         "iconName": "following",
-        "tags": "person, human, male, female, character, customer, individual, social",
+        "tags": "person, human, male, female, character, customer, individual, social, receive, message, request",
         "group": "interface",
         "codepoint": 61886
     },
     {
         "displayName": "Menu",
         "iconName": "menu",
-        "tags": "navigation, lines, list",
+        "tags": "navigation, lines, list, stack, hamburger, paragraph, document, section",
         "group": "interface",
         "codepoint": 62016
     },
     {
         "displayName": "Collapse all",
         "iconName": "collapse-all",
-        "tags": "arrows, chevron, reduce",
+        "tags": "arrows, chevron, reduce, smash, in between, up down",
         "group": "interface",
         "codepoint": 61790
     },
     {
         "displayName": "Expand all",
         "iconName": "expand-all",
-        "tags": "arrows, chevron, enlarge",
+        "tags": "arrows, chevron, enlarge, stretch, flex, volume",
         "group": "interface",
         "codepoint": 61855
     },
     {
         "displayName": "Intersection",
         "iconName": "intersection",
-        "tags": "circles, combine, cross",
+        "tags": "circles, combine, cross, venn diagram, overlap, link, intersect, union, inner join",
         "group": "action",
         "codepoint": 61957
     },
     {
         "displayName": "Blocked person",
         "iconName": "blocked-person",
-        "tags": "person, human, male, female, character, customer, individual, social, banned, prohibited",
+        "tags": "human, male, female, character, customer, individual, social, banned, prohibited, box remove, box minus",
```

what does "box" mean here? doesn't seem relevant

```suggestion
        "tags": "human, male, female, character, customer, individual, social, banned, prohibited",
```